### PR TITLE
Stub for mounting as typical docker user.

### PR DESCRIPTION
**Pull request by blaggacao**
See http://linux.die.net/man/8/mount.cifs
Stub for https://github.com/gondor/docker-volume-netshare/issues/14

### CODE REVIEW

File: netshare/drivers/cifs.go [hunk 1]
```diff
@@ -193,7 +193,11 @@ func (s cifsDriver) mountVolume(source, dest string, creds *cifsCreds) error {
 		opts.WriteString(fmt.Sprintf("sec=%s,", security))
 	}
 
-	opts.WriteString("rw ")
+	opts.WriteString("rw,")
+	opts.WriteString("uid=1000,")
```
Comment (blaggacao) [inline netshare/drivers/cifs.go]: I think those be better retrieved based on an actual check of the docker daemon if that's easily doable....

File: netshare/drivers/cifs.go [hunk 2]
```diff
@@ -193,7 +193,11 @@ func (s cifsDriver) mountVolume(source, dest string, creds *cifsCreds) error {
 		opts.WriteString(fmt.Sprintf("sec=%s,", security))
 	}
 
-	opts.WriteString("rw ")
+	opts.WriteString("rw,")
```
Comment (blaggacao) [inline netshare/drivers/cifs.go]: I think ro option should be switchable, out of stupidity I erased my development repo content which is a really cool learning effect.. :dancer:  However I bet I'll forget this is possible in about 3 to 4 months...

Comment (blaggacao) [inline netshare/drivers/cifs.go]: I beg your pardon, IIRC the docker volume mount is capable doing this at the second step... this then would be just an error at debugging and playing around time...

Comment (gondor) [inline netshare/drivers/cifs.go]: Are you referring to this package within Docker?  https://github.com/docker/docker/tree/master/pkg/mount

Comment (blaggacao) [inline netshare/drivers/cifs.go]: I think so. Which would translate into `docker inspect`'s:
```
Mounts": [
    {
        "Name": "fac362...80535",
        "Source": "/var/lib/docker/volumes/fac362...80535/_data",
        "Destination": "/webapp",
        "Driver": "local",
        "Mode": "",
        "RW": true,
        "Propagation": ""
    }
]
```
So it obviously handles the RW on this second step as well...
` docker run -v host/foo:/container/foo bar` which is an implicit
` docker run -v host/foo:/container/foo:rw bar`, but can be changed to
` docker run -v host/foo:/container/foo:ro bar`

**Comment by blaggacao** [comment]
The underlying issue went away.

**Pull request closed without merging**